apps/condo/bin/migrate-user-import-id-data-to-user-external-identity.js

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
Could you please rename your script according to `fingerprint` in `sender`.
I've suggested fingerprint name and name of this script I'm suggesting is "migrate-user-import-data-to-external-Identity"